client/src/reviews/ReviewsList.tsx

```diff
@@ -1,36 +1,59 @@
-import React, { ReactElement, useContext } from 'react';
+import React, { ReactNode, useContext, useEffect, useState } from 'react';
 import Contexts from '../contexts/Contexts';
 import ReviewSort from './ReviewSort';
 import ReviewTile from './ReviewTile';
 
 const ReviewsList: React.FC = () => {
-  // const [displayReviews, setDisplayReviews] = useState(false);
-  // const [numDisplayed, setNumDisplayed] = useState(2);
-  // const [filterType, setFilterType] = useState('relevant');
-
-
   const { setModalContent } = useContext(Contexts.ModalContext) || {};
   const { reviews } = useContext(Contexts.ReviewsContext) || {};
-  const child = <div>CHILD EXAMPLE</div>;
 
+  const [displayedReviews, setDisplayReviews] = useState<ReactNode[]>([]);
+  const [numDisplayed, setNumDisplayed] = useState(2);
+
+  const child = <div>CHILD EXAMPLE</div>;
 
-  const onClick = () => {
+  const addReview = () => {
     setModalContent?.call(null, child);
   };
 
+  const moreReviews = () => {
+    if (reviews?.results) {
+      if (numDisplayed === 14 || (reviews.results.length - 1) === numDisplayed) {
```

Comment:
I feel like this section could be a lot cleaner.  It seems like the first if statement should be nested within the second one, instead of being an `else if`. That might also allow you to reuse the scrolling behavior